spec/unit/jobs/v3/delete_service_instance_job_spec.rb

```diff
@@ -171,17 +192,6 @@ module V3
           expect(subject.restart_on_failure?).to eq(true)
         end
       end
-
-      describe '#pollable_job_state' do
-        it 'returns POLLING' do
-          expect(subject.pollable_job_state).to eq(PollableJobModel::POLLING_STATE)
-        end
-
-        it 'returns PROCESSING when the request has failed' do
-          subject.instance_variable_set(:@request_failed, true)
-          expect(subject.pollable_job_state).to eq(PollableJobModel::PROCESSING_STATE)
-        end
-      end
```

Comment:
I switched this to be a bit more behavioral / triggered off of how perform works.